modules/appeals_api/lib/appeals_api/central_mail_updater.rb

```diff
@@ -64,7 +79,7 @@ def parse_central_mail_response(raw_response)
     end
 
     def update_appeal_status!(appeal, status)
```

Comment:
with the method chain `status.status`, it's not immediately obvious what this object being passed in. Is there another name for it?

Reply:
Yeah, that is a good point. Thanks!